includes/extra_configures/enable_error_logging.php

```diff
@@ -52,18 +99,30 @@ function zen_debug_error_handler ($errno, $errstr, $errfile, $errline)
 
 /**
  * The path where the debug log file will be located
- * Default value is: DIR_FS_LOGS . '/myDEBUG-999999-00000000.log'
- * ... which puts it in the /logs/ folder:   /logs/myDEBUG-999999-00000000.log  (where 999999 is a random number, and 00000000 is the server's timestamp)
+ * Default value is: DIR_FS_LOGS . '/myDEBUG-yyyymmdd-hhssmm-xxxxx.log'
+ * ... which puts it in the /logs/ folder:   /logs/myDEBUG-yyyymmdd-hhiiss-xxxxx.log
+ *     where:
+ *      - yyyy .... is the 4-digit year
+ *      - mm ...... is the 2-digit month
+ *      - dd ..... is the 2-digit day-of-month
+ *      - hh ..... is the 2-digit hour
+ *      - ii ..... is the 2-digit minute
+ *      - ss ..... is the 2-digit second
+ *      - xxxxx ... is the time in milliseconds
+ *
  *    (or if you don't have a /logs/ folder, it will use the /cache/ folder instead)
  */
-$debug_logfile_path = DIR_FS_LOGS . '/myDEBUG-' . time() . '-' . mt_rand(1000,999999) . '.log';
+$log_prefix = (IS_ADMIN_FLAG) ? '/myDEBUG-adm-' : '/myDEBUG-';
+$log_date = new DateTime();
+$debug_logfile_path = DIR_FS_LOGS . $log_prefix . $log_date->format('Ymd-His-u') . '.log';
+unset($log_prefix, $log_date);
 
 /**
  * Error reporting level to log
  * Default: E_ALL ^E_NOTICE
  */
 $errors_to_log = (version_compare(PHP_VERSION, 5.3, '>=') ? E_ALL & ~E_DEPRECATED & ~E_NOTICE : version_compare(PHP_VERSION, 5.4, '>=') ? E_ALL & ~E_DEPRECATED & ~E_NOTICE & ~E_STRICT : E_ALL & ~E_NOTICE);
-
+$errors_to_log = -1;
```

Comment:
Did you mean to set this to `-1` ? If so, then the line above it is unneeded.

Reply:
Nope, debug code.  See update.